### PR TITLE
[PERF] base_vat: avoid checking vat validity with vies on import

### PR DESCRIPTION
**Pull request by sts-odoo**
The vies check service may limit ip addresses when:
- checking too many vat numbers per day
- checking the same vat number multiple times
- checking too many invalid number

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/base_vat/models/res_partner.py [hunk 1]
```diff
@@ -821,10 +821,16 @@ def create(self, vals_list):
             if values.get('vat'):
                 country_id = values.get('country_id')
                 values['vat'] = self._fix_vat_number(values['vat'], country_id)
-        return super(ResPartner, self).create(vals_list)
+        res = super().create(vals_list)
+        if self.env.context.get('import_file'):
+            res.env.remove_to_compute(self._fields['vies_valid'], self)
```
Comment (lobaudoux) [inline addons/base_vat/models/res_partner.py:826]: I think it should be:
```suggestion
            res.env.remove_to_compute(self._fields['vies_valid'], res)
```

**Review by lobaudoux — COMMENTED** [review]
It looks like you forgot the module name in the commit header.

**Review by lobaudoux — APPROVED** [review]
LGTM

**Comment by sts-odoo** [comment]
@robodoo r+

**Comment by lobaudoux** [comment]
@robodoo r+

**Pull request closed without merging**